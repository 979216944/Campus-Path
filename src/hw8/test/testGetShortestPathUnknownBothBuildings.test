# Verify that the program prints two unknown buildings
# with unknown origin and destination building.

# get shortest path with both buildings unknown
r
unknown1
unknown2

# quit
q