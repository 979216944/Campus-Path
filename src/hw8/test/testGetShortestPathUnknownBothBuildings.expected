Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Verify that the program prints two unknown buildings
# with unknown origin and destination building.

# get shortest path with both buildings unknown
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: unknown1
Unknown building: unknown2

Enter an option ('m' to see the menu):
# quit
